README.md

```diff
@@ -0,0 +1,9 @@
+# Global Public Inclusive Infrastructure: Code Repositories
+
+Repository for the
+[public-facing GitHub pages of GPII](http://gpii.github.io/).
+
+## Licence and Copyright
+
+Licence: Creative Commons Attribution-ShareAlike 4.0 International [CC BY-SA 4.0](LICENCE.html).
```

Comment:
I think it would make sense to specify here that this page is CC, but that the rest of the repos have other licencelisted within the individual repos